.github/run_aws_mocktests.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
```

Comment:
Why run the regular integration tests on GitHub actions?  Can we just leave that technical debt behind and move to the molecule tests?  We're still running the integration tests on GitHub for now so if something isn't caught by the molecule tests but is caught by the integration tests, we'll find out